modules/apps/foundation/frontend-theme/responsive-boat-theme/src/js/main.js

```diff
@@ -0,0 +1,52 @@
+/* This function gets loaded when all the HTML, not including the portlets, is loaded. */
+AUI().ready(function() {}); /******************** NOT USED ********************/
+
+/*	This function gets loaded after each and every portlet on the page.
+	portletId: the current portlet's id
+	node: the Alloy Node object of the current portlet
+*/
+Liferay.Portlet.ready(function(portletId, node) {}); /******************** NOT USED ********************/
+
+/* This function gets loaded when everything, including the portlets, is on the page. */
+Liferay.on('allPortletsReady', myPageJS); /******************** USED ********************/
+
+
+
+// Function Containing All of My Event Handlers
+function myPageJS() {
+	let navBar = document.getElementById('navigation'),
```

Comment:
We should rename the JS variables to match the node that it is selecting.

Ex.
```
navigation.
```

Reply:
Each JS variable should be defined on its own line with their own signal.